src/components/ViewDialog/ViewDialog.tsx

```diff
@@ -30,10 +30,10 @@ export const ViewDialog = ({ row }: ViewDialogProps) => {
           </div>
         );
       }
-
       return (
         <div key={field} className="break-all">
-          <span className="text-gray-400">{field}:</span> {value as React.ReactNode}
+          <span className="text-gray-400">{field}:</span>{' '}
+          {typeof value === 'boolean' ? value.toString() : (value as React.ReactNode)}
```

Comment:
Not a real issue but just curious. Why do we need to cast it here as ReactNode?

Reply:
without `ReactNode` it throws an error
`TS2322: Type unknown is not assignable to type ReactNode`